### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by samjwu**
encompasses dependabot alerts 567-569

closes #567, closes #568, closes #569

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -4,9 +4,15 @@
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+import subprocess
 from rocm_docs import ROCmDocs
 
-docs_core = ROCmDocs("hipBLAS Documentation")
+getVersionCmd = r'sed -n -e "s/^.*VERSION_STRING.* \"\([0-9\.]\{1,\}\).*/\1/p" ../CMakeLists.txt'
```
Comment (saadrahim) [inline docs/conf.py:10]: this should be in rocm-docs-core. But please test here first if you want

**Review by saadrahim — DISMISSED** [review]

File: docs/conf.py (hunk 1 above)
Comment (samjwu) [reply docs/conf.py]: reverted. 
44521d6

**Comment by rkamd** [comment]
disclaimer.rst needs to be removed.

**Review by rkamd — APPROVED** [review]
LGTM

**Pull request merged**